mllib/src/test/java/org/apache/spark/mllib/tree/JavaDecisionTreeSuite.java

```diff
@@ -95,6 +97,14 @@ public void runDTUsingStaticMethods() {
 
     DecisionTreeModel model = DecisionTree$.MODULE$.train(rdd.rdd(), strategy);
 
+    // java compatibility test
+    JavaRDD<Double> predictions = model.predict(rdd.map(new Function<LabeledPoint, Vector>() {
+      @Override
+      public Vector call(LabeledPoint v1) throws Exception {
```

Comment:
Nit: no "throws Exception". Can you omit the types above and write new Function<>?